app/src/main/java/com/codesoom/assignment/application/CatToyService.java

```diff
@@ -0,0 +1,49 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.CatToy;
+import com.codesoom.assignment.domain.CatToyRepository;
+import com.codesoom.assignment.exception.CatToyNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+/**
+ * 고양이 장난감을 관리한다.
+ */
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class CatToyService {
+    private final CatToyRepository catToyRepository;
+
+    public List<CatToy> findAll() {
+        return catToyRepository.findAll();
+    }
+
+    public CatToy findById(Long id) {
```

Comment:
primitive 타입을 써도 괜찮지 않을까요?
```suggestion
    public CatToy findById(long id) {
```

Reply:
CatToyRepository에서 제네릭타입을 `<CatToy, Long>`으로 선언해주면서 아이디의 타입을 wrapper class로 선언 해줬기에
불필요한 boxing/unboxing을 막기 위해 primitive타입 대신  wrapper class type으로 선언해줬는데, primitive 타입으로 해줬을 때 더 좋은점이 있을까요?

Reply:
아까 슬랙으로 대화했으니 코멘트 패스할게요.